tests/gpu-tests/test_vllm_audio.py

```
@@ -0,0 +1,85 @@
+
+import pytest
+from utils import require_env_var
+
+
+@pytest.mark.gpu
+def test_vllm_audio_generation():
+    """Integration test: Generate with vLLM server using audio input."""
+    model_path = require_env_var("NEMO_SKILLS_TEST_HF_MODEL")
+    model_type = require_env_var("NEMO_SKILLS_TEST_MODEL_TYPE")
+
+    output_dir = f"/tmp/nemo-skills-tests/{model_type}/vllm-audio-generation"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/nemo-skills-tests/"

(S108)